tests/test_resource.py

```diff
@@ -263,63 +253,60 @@ def make_cors_handler(self, origins: List[str] = None, req_origin: str = None) -
         cors_handler = CORSResource(an_event, ["GET", "POST"], origins=origins)
         return cors_handler
 
-    def test_cors_origin_headers_from_env_are_correct_1(self):
-        assert self.make_cors_handler().resp_headers_json[ALLOW_ORIGIN_HEADER] == ORIGIN_LOCALHOST
-
-    def test_cors_origin_headers_from_env_are_correct_2(self):
-        headers = self.make_cors_handler(req_origin=ORIGIN_EXAMPLE).resp_headers_json[
+    @pytest.mark.parametrize(
+        "req_origin, orig_outcome",
+        [
+            (None, ORIGIN_LOCALHOST),
+            (ORIGIN_EXAMPLE, ORIGIN_EXAMPLE),
+            ("invalid_origin", ORIGIN_LOCALHOST),
+        ],
+    )
+    def test_cors_origin_headers_from_env_are_correct(self, req_origin, orig_outcome) -> None:
+        headers = self.make_cors_handler(req_origin=req_origin).resp_headers_json[
             ALLOW_ORIGIN_HEADER
         ]
-        assert headers == ORIGIN_EXAMPLE
+        assert headers == orig_outcome
```

Comment:
Please improve naming around this test. Why would `headers` be equal to `orig_outcome`? What is `orig_outcome`? Those are questions I asked reading it for the first time 😬